package.json

```diff
@@ -25,18 +25,16 @@
 		"abi/"
 	],
 	"devDependencies": {
-		"@defi-wonderland/smock": "^2.2.0",
-		"@nomicfoundation/hardhat-chai-matchers": "^1.0.2",
-		"@nomicfoundation/hardhat-network-helpers": "^1.0.3",
-		"@nomiclabs/hardhat-ethers": "^2.1.1",
+		"@defi-wonderland/smock": "^2.3.4",
```

Comment:
There is a conflict peer dependency between smock and the hardhat-ethers, causes the failing of CI.
Do we have plan to solve this issue?

Reply:
yeah i know. figuring out a workaround. it was working perfect locally. but turned out the conflicts ain't resolved when installed from scratch. apart from being slow, this is another annoying issue with `npm` - unresolved dependencies. in the Mynt repo the same setup but with yarn worked like a charm.

Reply:
resolved - no CI failures